src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -18,13 +19,14 @@ export default class SaveInProgressIntro extends React.Component {
         const savedAt = this.props.lastSavedDate
           ? moment(this.props.lastSavedDate)
           : moment.unix(savedForm.last_updated);
+        const expirationDate = moment(savedAt).add(30, 'days');
```

Comment:
Copying from Slack, this _should_ be available in `savedForm.expires_at` if `savedForm` references the `metadata` of the saved claim.